src/bsoncxx/test/CMakeLists.txt

```diff
@@ -70,7 +70,14 @@ target_link_libraries(bsoncxx_test_properties_with_main INTERFACE bsoncxx::test_
 add_library(bsoncxx::test_properties_with_main ALIAS bsoncxx_test_properties_with_main)
 
 target_link_libraries (test_bson PRIVATE bsoncxx::test_properties_with_main)
-add_test(NAME bson COMMAND test_bson)
+
+set(test_args "")
+list(APPEND test_args
+    --reporter compact
+    --allow-running-no-tests
+)
+
+add_test(NAME bson COMMAND test_bson ${test_args})
```

Comment:
Is the `test_args` variable really needed? Can arguments be inlined into the `add_test` call?

Reply:
It is just for consistency with mongocxx tests. I suppose it can be inlined for now until more bsoncxx test executables are added (if any).